orchestrator/services/tasks.py

```diff
@@ -116,3 +122,20 @@ def resume_task(pid: UUID, user_inputs: Optional[List[State]], user: str) -> Opt
     def resume_workflow(pid: UUID, user_inputs: Optional[List[State]], user: str) -> Optional[UUID]:
         local_logger.info("Resume workflow", pid=pid)
         return resume_process(pid, user_inputs=user_inputs, user=user)
+
+
+class CeleryJobWorkerStatus(WorkerStatus):
+    def __init__(self) -> None:
+        super().__init__(executor_type="celery")
+        if not _celery:
+            logger.error("Can't create CeleryJobStatistics. Celery is not initialised.")
+            return
+
+        inspection: Inspect = _celery.control.inspect()
+        stats = inspection.stats()
+        logger.info("Inspection stats")
+        self.number_of_workers_online = len(stats)
+        self.number_of_queued_jobs = sum(len(tasks) for w, tasks in inspection.scheduled().items()) + sum(
```

Comment:
Or maybe just a small local helper function that does the sum to increase readability a bit here

Reply:
Done